src/main/java/marquez/db/JobVersionDao.java

```diff
@@ -106,17 +109,32 @@ default void insertAndUpdate(JobVersionRow row) {
           + "  ON (job_uuid = jv.job_uuid AND j.name = :jobName)"
           + "INNER JOIN namespaces AS n "
           + "  ON (j.namespace_uuid = n.uuid AND n.name = :namespaceName) "
+          + "INNER JOIN job_contexts AS jc "
+          + "  ON (job_context_uuid = jc.uuid) "
           + "ORDER BY created_at DESC "
           + "LIMIT 1")
-  Optional<JobVersionRow> findLatest(String namespaceName, String jobName);
+  Optional<ExtendedJobVersionRow> findLatest(String namespaceName, String jobName);
 
   @SqlQuery(
-      "SELECT j.* FROM jobs AS j "
+      "SELECT j.uuid AS job_uuid, j.namespace_uuid, jv.*, jc.uuid AS job_context_uuid, jc.context "
+          + "ARRAY(SELECT dataset_uuid "
```

Comment:
I assume the j.uuid is unique in the query result, otherwise the Array might get mixed up with multiple jobs.

Reply:
> I assume the j.uuid is unique in the query result, otherwise the Array might get mixed up with multiple jobs.

Yep, ` j.uuid` is unique!